ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumption.scala

```diff
@@ -363,6 +363,7 @@ object KeyValueConsumption {
             recordTime,
             rejectionEntry,
             errorVersionSwitch,
+            None, // Not available for historical entries
```

Comment:
What about the future entries? Can we pass an `Option` here?

Reply:
So this is not valid anymore because this flow will not have new entries. 
This is for the out of time bounds, and we no longer mix deduplication there

Reply:
Got it, thanks!